### PR TITLE
23/24 - bug fix responsiveness of all FE pages including Navbar

### PR DESCRIPTION
**Pull request by zhunan123**
https://github.com/user-attachments/assets/deb6d9e1-4731-4691-9b93-f88cc3fbcef7

https://github.com/user-attachments/assets/50477c88-fadc-4a6c-95d2-9d3a59e41ea9

https://github.com/user-attachments/assets/2bd336e0-2a9c-4dc8-8c34-e05bd3b255ad

### CODE REVIEW

**Review by lemonshark05 — CHANGES_REQUESTED** [review]
Everything else is fine, but I want to know if the Navbar could be reduced to the smallest version and retain the text 'Impact Match' next to the logo. Impact Match would be better.

File: frontend/src/Signup.js [hunk 1]
```diff
@@ -2,131 +2,131 @@ import React, { useState } from 'react';
...
+                                Organization
+                            </button>
+                        </div>
+                        <img src={icon} alt="Impact Match Logo" className="w-32 h-32 mb-4" />
+                        <h1 className="text-2xl font-inter">Impact Match</h1>
+                    </div>
+                    {/* Right Section */}
+                    <div className="w-full md:w-2/3 p-10 flex flex-col items-center justify-center">
+                    <h2 className="text-2xl mb-6 text-gray-600 text-center font-inter">Create New Account</h2>
+                    <p className="text-sm mb-28 text-gray-400 text-center">Already Registered? <button className="text-lightBlue">Login</button></p>
```
Comment (jcai28) [inline frontend/src/Signup.js:37]: Make sure change this button to a link

File: frontend/src/Signup.js [hunk 2]
```diff
@@ -2,131 +2,131 @@ import React, { useState } from 'react';
...
+                                    >
+                                        Sign Up
+                                    </button>
+                                </div>
+                                <div className="flex justify-center mt-4">
+                                    <button className="min-w-[180px] py-2 flex items-center justify-center bg-white text-black border border-gray-300 rounded-2xl text-sm">
+                                        <img src={googleIcon} alt="Google Icon" className="w-5 h-5 mr-2" />
+                                        Sign in with Google
+                                    </button>
+                                </div>
```
Comment (jcai28) [inline frontend/src/Signup.js:126]: We decide to remove the google sign up feature

File: frontend/src/Login.js [hunk 3]
```diff
@@ -23,77 +23,71 @@ const Login = () => {
...
-                        </Link>
-                    </p >
-                </div>
+        <div className="flex flex-col h-screen w-screen bg-white">
+            <div className="flex-grow flex flex-col justify-center items-center"> 
+                <div className="w-full max-w-md p-10 bg-white">
+                    <h2 className="text-2xl mb-6 text-gray-600 text-left font-bold">Sign In</h2>
+                    <p className="text-sm mb-6 text-gray-400 text-left">
+                        Or{' '}
+                        <Link to="/signup" className="text-lightBlue hover:underline">create an account</Link>
```
Comment (jcai28) [inline frontend/src/Login.js:32]: Cap this "create"

Comment (jcai28) [reply frontend/src/Login.js:32]: let the form and title align

**Review by jcai28 — COMMENTED** [review]
The responsive design could be refined later, current version is working to me.

File: frontend/src/Login.js (hunk 3 above)
Comment (zhunan123) [reply frontend/src/Login.js:32]: implemented as sponsor's design.

File: frontend/src/Signup.js (hunk 2 above)
Comment (zhunan123) [reply frontend/src/Signup.js:126]: done, removed google sign in

File: frontend/src/Signup.js (hunk 1 above)
Comment (zhunan123) [reply frontend/src/Signup.js:37]: done, changed to link.

**Review by lemonshark05 — APPROVED** [review]

**Pull request merged**